components/leave_private_channel_modal/leave_private_channel_modal.tsx

```diff
@@ -6,18 +6,17 @@ import {FormattedMessage} from 'react-intl';
 
 import {Channel} from 'mattermost-redux/types/channels';
 
-import ModalStore from 'stores/modal_store.jsx';
-import Constants from 'utils/constants';
 import ConfirmModal from 'components/confirm_modal';
 
 type State = {
     show: boolean;
-    channel?: Channel;
 };
 
 type Props = {
+    channel?: Channel;
```

Comment:
Why make this optional?

Reply:
actually yeah it shouldnt be optional, changed it to is required